django/utils/timezone.py

```diff
@@ -295,13 +295,15 @@ def template_localtime(value, use_tz=None):
 
 # Utilities
 
-def localtime(value, timezone=None):
+def localtime(value=None, timezone=None):
```

Comment:
Yes, I know. I was suggesting that to parallel `localdate`, `localtime` with no `value` when `USE_TZ=False` should return `datetime.datetime.now()` (the naive current local time).

I think this may still be a valuable parallel, even though calling `localtime` with a naive `value` does not work. But I don't feel strongly about it. Curious what @aaugustin thinks.